pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -1143,6 +1154,25 @@ func Test_Run_Positive_VolumeFSResizeControllerAttachEnabled(t *testing.T) {
 				defer close(stoppedChan)
 				reconciler.Run(stopChan)
 			}()
+			// uncertainTest means that in actualstate, the volume mount state is uncertain
+			if tc.uncertainTest {
+				waitErr := retryWithExponentialBackOff(testOperationBackOffDuration, func() (done bool, err error) {
+					mounted, _, err := asw.PodExistsInVolume(podName, volumeName)
+					if mounted || err != nil {
+						return false, nil
+					}
+					for _, mountedVolume := range asw.GetPossiblyMountedVolumesForPod(podName) {
+						if mountedVolume.VolumeName == volumeName {
+							return true, nil
+						}
+					}
+					return false, nil
+				})
+				if waitErr != nil {
+					t.Fatalf("Volume resize should succeeded %v", waitErr)
+				}
+				return
+			}
 			waitForMount(t, fakePlugin, volumeName, asw)
```

Comment:
Does this test belong here or in `Test_UncertainVolumeMountState` tests? I think that - it rather belongs there, rather than here.

Reply:
right, I moved the test case, I think it is better.